README.md

```diff
@@ -1,4 +1,6 @@
 # FFAAS
+145068ff-e155-4c75-ab9f-9dbef07c7ef6
```

Comment:
Those were the deployID during the Live. They should probably not stay in the README.